stanford_profile.profile

```diff
@@ -167,6 +168,28 @@ function stanford_profile_node_access(NodeInterface $node, $op, AccountInterface
   return AccessResult::neutral();
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function stanford_profile_form_menu_edit_form_alter(array &$form, FormStateInterface $form_state) {
+  $read_only = Settings::get('config_readonly', FALSE);
+  if (!$read_only) {
+    return;
+  }
+
+  // If the form is locked, hide the config you cannot change from users without
+  // the know how.
+  $access = \Drupal::currentUser()->hasPermission('Administer menus and menu items');
+  $form['label']['#access'] = $access;
+  $form['description']['#access'] = $access;
+  $form['id']['#access'] = $access;
```

Comment:
Not really needed but :man_shrugging:  :neutral_face:

Reply:
It did show up when the label was hidden. I thought it was extra at first too.